### PR TITLE
feature : wishlist-cart

### PR DESCRIPTION
**Pull request by MurtazaBagwala07**
user can add item to wishlist
user can remove item from wishlist
user can add item to cart
user can remove item from cart
dynamic data shown in header (number of items in cart and wishlist)
price card shows dynamic values acc to items in cart

### CODE REVIEW

**Review by shrey27 — APPROVED** [review]
Code structure is good
Logic seems to be working
PR is good to merge

File: jersea/src/components/index.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
 export {ProductCard} from './Products/ProductCard/ProductCard.jsx'
 export {ProductFilter} from './Products/ProductFilter/ProductFilter.jsx'
 export {ProductList} from './Products/ProductList/ProductList.jsx'
-export {Header} from './Header/Header.jsx'
\ No newline at end of file
+export {Header} from './Header/Header.jsx'
```
Comment (ramalingaa) [inline jersea/src/components/index.js:4]: Try using index file for reusable components

**Review by ramalingaa — APPROVED** [review]
Code quality is good.
Logic is working
PR can be merged

File: jersea/src/components/index.js (hunk 1 above)
Comment (MurtazaBagwala07) [reply jersea/src/components/index.js:4]: sure will do it

**Pull request merged**